License

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2018 Government Digital Service
```

Comment:
See https://github.com/govau/cloud.gov.au/blob/master/LICENSE#L3 - copyright entity needs to be updated. You can copy that LICENSE in directly. Also could you please update the name to uppercase LICENSE?

Reply:
Changed